nnoir-onnx/test/test_matmul.py

```diff
@@ -37,7 +37,6 @@ def create_onnx(self) -> onnx.ModelProto:
     MatMulTester({"x": x}, outputs).run()
 
 
-@pytest.mark.xfail()
```

Comment:
Because of the improvement, we can remove the mark of test-fail.
https://github.com/Idein/nnoir/pull/101/files#r592835095

Reply:
reverted